code/src/components/Form.js

```diff
@@ -0,0 +1,115 @@
+import React, {useState} from 'react'
+
+import OrderBtn from './OrderBtn'
+import SelectSize from './SelectSize'
+import Name from './Name'
+import Email from './Email'
+import Hero from './HeroImage'
+import Phone from './Phone'
+import Amount from './Amount'
+import Checkbox from './Checkbox'
+import Radio from './Radio'
+import Range from './RangeSlider'
+
+const Form = () => {
+    const [sumHidden, setsumHidden] = useState(true);
+    const [Size, setSize] = useState("");
+    const [myamount, setmyamount] = useState("");
+    const [Color, setColor] = useState();
+    const [myname, setmyname] = useState("");
+    const [myemail, setmyemail] = useState("");
+    const [wantsNewsletter, setwantsNewsletter] = useState("");
+    const [myphone, setmyphone] = useState("");
+    const [Rng, setRng] = useState("No Opinion");
+
+    const SelectedRange = (e) => {
+        console.log(e.target)
+        setRng(e.target.value)
+    }
+
+    const SelectPhoneNumber = (e) => {
+        console.log(e.target.value)
+        setmyphone(e.target.value)
+    }
+
+    const SelectCheckbox = (e) => {
+        console.log(e.target.value)
+        setwantsNewsletter(e.target.value)
+    }
+
+    const SelectEmail = (e) => {
+        console.log(e.target.value)
+        setmyemail(e.target.value)
+    } 
+
+    const SelectName = (e) => {
+        console.log(e.target.value)
+        setmyname(e.target.value)
+    }
+
+    const SelectAmount = (e) => {
+        console.log(e.target.value)
+        setmyamount(e.target.value)
+    }
+
+    const SelectS = (e) => {
+        console.log(e.target.value)
+        setSize(e.target.value)
+    }
+
+    const SelectColor = (e) => {
+        console.log(e.target.value)
+        setColor(e.target.value)
+    }
+
+    const OnSumChange = (event) => {
+        event.preventDefault()
+        setsumHidden (false)
+    }
+
+    if (sumHidden === true) {
```

Comment:
You have in the state declared sumHidden to "true". Therefore you could ad if (sumHidden) instead of if (sumHidden === true). Just a thought 👍